backend/reservations/checks.py

```diff
@@ -1,13 +1,62 @@
 from django.core.checks import Error, Warning, Info, register
 from reservations.models import Room, Guest
 
+def ticket_chain(p_guest, p_chain=[]):
+    a_chain = [p_guest] + p_chain
+
+    if not p_guest.transfer or p_guest.transfer == '':
+        return a_chain
+
+    try:
+        a_guest = Guest.objects.get(ticket=p_guest.transfer)
+    except Guest.DoesNotExist:
+        return a_chain
+
+    return ticket_chain(a_guest, a_chain)
+
 @register(deploy=True)
 def guest_drama_check(app_configs, **kwargs):
     errors = []
     guests = Guest.objects.all()
+
+
     for guest in guests:
         if guest.jwt == '' and guest.can_login:
-            errors.append(Error(f"Guest {guest.email} has an empty jwt field!"))
+            errors.append(Warning(f"Guest {guest.email} has an empty jwt field!",
+                                  hint="Reset password via ux or user_edit"),
+                          obj=guest)
+
+        if guest.ticket and Guest.objects.filter(ticket = guest.ticket).count() > 1:
+            errors.append(Error(f"Guest {guest.email}, ticket {guest.ticket} shared with other users"))
+
+        guest_transfer_chain = ticket_chain(guest)
+        chain_tail = [x for x in guest_transfer_chain if x.transfer == '']
+        if len(chain_tail) == 0:
+            errors.append(Error(f"Transfer chail tail not found {[x.ticket for x in guest_transfer_chain]}",
+                          hint="Unable to find guest record w/o transfer in this chain." \
+                                "Manually reconcile against SP exports.", obj=guest))
+
+        for chain_guest in guest_transfer_chain:
+            if chain_guest.transfer == '' \
+               and chain_guest.ticket != '' \
+                and chain_guest.hotel is not None \
+                and chain_guest.room_number is not None:
+                try:
+                    chain_room = Room.objects.get(name_hotel=chain_guest.hotel, number=chain_guest.room_number)
+                    if chain_room.is_placed and chain_guest.ticket != chain_room.sp_ticket_id:
+                        errors.append(Warning(f"Room {chain_room.name_take3} {chain_room.number} ticket {chain_room.sp_ticket_id}" \
+                                              f"does not match guest {guest.name}, ticket {guest.ticket}",
+                                      hint="Manual reconciliation? Good luck, starfighter.",
+                                      obj=guest))
+                except Room.DoesNotExist:
```

Comment:
I think this is mostly a style thing  but I would move the DoesNotExist exception handler to right after the code that could raise this exception, Room.objsects.get I assume. if the room does not exist there is no point to checking .is_placed or the other things. 
Also tho risk seems low and this is already written so00oo.